metadata/ComputerVision__1__metadata.json

```diff
@@ -14,8 +14,8 @@
   "projectId": "[project-id]",
   "retrainable": false,
   "stagingUri": "[staging-uri]",
-  "projectName": "UiPath Document Understanding",
-  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",

```

Comment:
I am doubtful about Backend server bit ... Should we be mentioning in projectDescription ?
cc @vivekjhauipath

Reply:
Metadata comes from individual teams/PMs.
@SahilChaudhary25 did we get this from Stefan/Cosmin?

Reply:
It is given by cosmin